q2_taxa/_barplot_visualizer/src/lib/ColorControls.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+    import { sampleManager } from "./../classes/sampleManager.svelte";
+    import { Taxon } from "./../classes/taxonomy.svelte";
+
+    function handleColorSchemeChange(event: Event) {
+        sampleManager.render();
+    }
+
+    function handleRemoveColor(taxon: Taxon) {
+        return (event: Event) => {
+            sampleManager.colors.removeCustomColor(taxon);
+
+            sampleManager.render();
+        };
+    }
+
+    let show = $state(false);
```

Comment:
Like with the other one, is this vestigial, or is it being used sneakily somewhere somehow?